opensearch/src/test/java/org/opensearch/sql/opensearch/response/OpenSearchAggregationResponseParserTest.java

```diff
@@ -36,12 +36,7 @@ class OpenSearchAggregationResponseParserTest {
   /** SELECT MAX(age) as max FROM accounts. */
   @Test
   void no_bucket_one_metric_should_pass() {
-    String response =
-        "{\n"
-            + "  \"max#max\": {\n"
-            + "    \"value\": 40\n"
-            + "  }\n"
-            + "}";
+    String response = "{\n" + "  \"max#max\": {\n" + "    \"value\": 40\n" + "  }\n" + "}";
```

Comment:
Can you keep it multiline or if not, simplify the string
```suggestion
    String response = "{ \"max#max\": { \"value\": 40 } }";
```

Reply:
removed `+`'s